model/vcs/metrics.yaml

```diff
@@ -0,0 +1,139 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: updowncounter
+    unit: "{change}"
+    stability: experimental
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.url.full
+        requirement_level: required
+  - id: metric.vcs.repository.change.time
+    type: metric
+    metric_name: vcs.repository.change.time
+    brief: 'The time distribution a change (pull request) has been in open state or that it took to go from open to another state.'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+      - ref: vcs.change.state
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_to_approval
+    type: metric
+    metric_name: vcs.repository.change.time_to_approval
+    brief: 'The amount of time it took a change (pull request) to go from open to approved'
+    instrument: histogram
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.url.full
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.count
+    type: metric
+    metric_name: vcs.repository.count
+    brief: 'The number of repositories in an organization'
+    instrument: gauge
```

Comment:
I think it should be an updowncounter since it's additive:

 https://github.com/open-telemetry/opentelemetry-specification/blob/ea3111b78ce72206f4ddeea728b5324df62878a5/specification/metrics/supplementary-guidelines.md#instrument-selection


> * I want to **measure** something (by reporting an absolute value):
>  * If the measurement values are [non-additive](#additive-property), use
>     an [Asynchronous Gauge](./api.md#asynchronous-gauge).
>   * If the measurement values are [additive](#additive-property):
>     * If the value is monotonically increasing - use an [Asynchronous
>       Counter](./api.md#asynchronous-counter).
>     * If the value is NOT monotonically increasing - use an [Asynchronous
>       UpDownCounter](./api.md#asynchronous-updowncounter).